src/v/cluster/controller_backend.cc

```diff
@@ -1919,6 +1926,12 @@ controller_backend::shutdown_partition(ss::lw_shared_ptr<partition> partition) {
           ntp, gr, partition->get_log_revision_id());
         // shutdown partition
         co_return co_await _partition_manager.local().shutdown(ntp);
+    } catch (seastar::gate_closed_exception&) {
```

Comment:
`const&` does it matter?

Reply:
I doubt it does TBH, as it's unused. But I'll change it at least for it to look nicer.